CMakeLists.txt

```diff
@@ -2,13 +2,13 @@
 # CMake configuration
 # ===========================================================================
 
-CMAKE_MINIMUM_REQUIRED (VERSION 3.14)
+CMAKE_MINIMUM_REQUIRED (VERSION 3.0)
 
 # ===========================================================================
 # Project configuration
 # ===========================================================================
 
-PROJECT (antaresXpansion VERSION 1.1.0)
+PROJECT (antaresXpansion VERSION 0.1.0)
```

Comment:
usually version numbers go up, not down. Right ?

Reply:
Yes but this was not the correct version. A application name change is considered.